modules/op-installing-pipelines-operator-using-the-cli.adoc

```diff
@@ -22,17 +21,17 @@ metadata:
   name: openshift-pipelines-operator
   namespace: openshift-operators
 spec:
-  channel:  <channel name> <1>
-  name: openshift-pipelines-operator-rh <2>
-  source: redhat-operators <3>
+  channel:  <channel_name> # <1>
+  name: openshift-pipelines-operator-rh # <2>
+  source: redhat-operators # <3>
   sourceNamespace: openshift-marketplace <4>
 ----
-<1> The channel name of the Operator. The `pipelines-<version>` channel is the default channel. For example, the default channel for {pipelines-title} Operator version `1.7` is `pipelines-1.7`. The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator.
+<1> Name of the channel to subscribe to. The `pipelines-<version>` channel is the default channel. For example, the default channel for {pipelines-title} Operator version `1.7` is `pipelines-1.7`. The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator.
 <2> Name of the Operator to subscribe to.
-<3> Name of the CatalogSource that provides the Operator.
+<3> Name of the `CatalogSource` that the Operator provides.
 <4> Namespace of the CatalogSource. Use `openshift-marketplace` for the default OperatorHub CatalogSources.
```

Comment:
```suggestion
<4> Namespace of the `CatalogSource` object. Use `openshift-marketplace` for the default OperatorHub catalog sources.
```

I know this is probably out of scope for this PR. But it would be best if you can consider making this change.